lib/Backend/IRBuilder.cpp

```diff
@@ -6615,6 +6624,36 @@ IRBuilder::BuildEmpty(Js::OpCode newOpcode, uint32 offset)
         }
         break;
 
+    case Js::OpCode::BeginBodyScope:
+        // This marks the end of a param socpe which is not merged with body scope.
+        // So we have to first cache the closure so that we can use it to copy the initial values for
+        // body syms from corresponding param syms (LdParamSlot). Body should get its own scope slot.
+        this->AddInstr(
```

Comment:
Are we guaranteed at this point that local scope slots and/or frame display are required?